members/srujan0404.md

```diff
@@ -0,0 +1,2 @@
+name: srujan
+github_handle: srujan0404
```

Comment:
Hey @srujan0404 this looks good but can you also describe the pr,
have a look at this https://github.com/SST-OSS/club-onboarding/pull/3 you will get an idea